schema/url/standalone-map/plugin-floating-boxplot.raml

```diff
@@ -0,0 +1,39 @@
+#%RAML 1.0 Library
+
+types:
+  # Type name determines the name of the Java class that will be generated.
+
+  FloatingBoxplotPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      config: FloatingBoxplotSpec
+
+  # think this always gives outliers and means, saves some user configuration?
```

Comment:
this is a great point. Do we need the full power of each viz in the map? To keep things simple, can we lean toward as little user configuration as possible? A general question